src/app/shared/doc-viewer/doc-viewer.ts

```diff
@@ -82,6 +84,14 @@ export class DocViewer implements OnDestroy {
     this._elementRef.nativeElement.innerHTML = rawDocument;
     this.textContent = this._elementRef.nativeElement.textContent;
 
+    // janky way to differentiate between document/standalone example and embedded example load
```

Comment:
The comment ("janky") leads me to believe there will be a follow up to address this in a better way. Is that the case and if so, should there be some TODO/bug to track it?